docs/01_manual.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Manual
+layout: home
+---
+
+Manual
+======
+
+TABLE OF CONTENTS
+-----------------
+
+- [SYNOPSIS](#synopsis)
+- [DESCRIPTION](#description)
+- [ARGUMENTS](#arguments)
+- [INSTALLATION AND CONFIGURATION](#installation-and-configuration)
+- [BUILDING FROM SOURCE](#building-from-source)
+
+
+SYNOPSIS
+--------
+
+    easy-deposit-agreement-generator [ -s ] <datasetID> <agreement-file>
+
+
+DESCRIPTION
```

Comment:
This section is copied from `easy-deposit-agreement-creator` and is not applicable here. Please modify the `DESCRIPTION` section, especially the first and second paragraph.

Reply:
unresolving this comment, since it still talks about _searching for a dataset that corresponds to the given `datasetID`..._ and about Fedora usage. That is not the intention of this tool, right?